integration/users/create_user_test.go

```diff
@@ -40,6 +40,7 @@ func TestCreateUser(t *testing.T) {
 
 	testCases := map[string]struct { //nolint:vet // for readability
 		payload    bson.D
+		hasCred    bool
```

Comment:
What does that field mean? I would assume "has credentials", but then there are cases with password and `hasCred` being `false`

Reply:
It meant "has credential" indeed. I was able to improve this, removing the variable.